Watchdog/Watchdog.ino

```diff
@@ -27,34 +27,48 @@ void loop() {
     
     //NOT STARTED YET
     case 0:
+      currTime = millis();
       //check for change in hearbeat
       heartBeat = digitalRead(inPin);
       if (heartBeat != lastBeat){ changeCounter++; }
-      totalCounter++;
+      
```

Comment:
Create a function that does the heartbeat checking. Something like:

``` 
void detectHeartbeat(){
    heartBeat = digitalRead(inPin);
    if(heartBeat != lastBeat) {
        changeCounter++;
    }
    lastbeat = heartBeat;
}
```
Since everything is a global it should work just fine